### DISCUSSION TITLE
Kafka Api port

### DISCUSSION THREAD

**Discussion by dg-korolev** [ydb-platform/ydb#4666] [Q&A] [answered]
В документации к Kafka API приводится пример:
https://ydb.tech/docs/en/reference/kafka-api/examples
В примере используется порт 9093, не нахожу в документации более информации о том как открыть этот порт для работы с ydb topics по протоколу kafka?

**Comment by zinal** [comment, marked as answer]
Настройку можно сделать в динамической конфигурации: https://github.com/ydb-platform/ydb/blob/24.3.9/ydb/core/protos/config.proto#L1815

Добавлю Issue, чтобы описать порядок настройки в документации.
